File: 백준/Bronze/1681. 줄 세우기/줄 세우기.c
#include <stdio.h>
#include <string.h>
#include <stdlib.h>

int main(void) { 
   int a, b, index = 0, current = 1, flag;
   char temp[8];
   scanf("%d %d", &a, &b);

   while(1){
      flag = 1;
      sprintf(temp, "%d", current);
      for(int k = 0; k < strlen(temp); k++){
         if(temp[k] == b + '0'){
            flag = 0;
            break;
         }
      }
      if(flag){
         index += 1;
      }   

      if(index == a)   break;
      current += 1;
   }

   printf("%d", current);   

   return 0;
}

File: 백준/Bronze/1681. 줄 세우기/줄 세우기.py
a, b = map(int, input().split())

current = 1
index = 0
while(True):
    flag = True
    
    string = str(current)
    if str(b) in string:
        flag = False
    
    if flag:
        index += 1
    
    if index == a:
        break
        
    current += 1

print(current)